sormas-e2e-tests/src/test/resources/features/sanity/api/Person.feature

```diff
@@ -0,0 +1,6 @@
+@Sanity @Person
+Feature: Check Person RestApi endpoints
+
+  Scenario: Create a new event
+    Given API: I receive all person ids
+    Given API: I receive the person
```

Comment:
The test doesn't have any assert. Don't you need an assert to check that you actually received the person? To verify something specific for that person from the response(first name for eg)?

Reply:
This is only to introduce the concept of API testing. You are right there is no assert at this moment